.github/workflows/ci.yaml

```diff
@@ -752,22 +752,40 @@ jobs:
         )
     env:
       IMAGE_REPOSITORY: ghcr.io/${{ github.repository_owner }}
+      TRENTO_DOMAIN: ${{ secrets.TRENTO_DEMO_IP }}
+      TRENTO_NAMESPACE: ${{ secrets.TRENTO_NAMESPACE }}
```

Comment:
Hey @rtorrero I don't see a `TRENTO_NAMESPACE` secret here https://github.com/trento-project/web/settings/secrets/actions

Is it intentional?

Reply:
added!